client/me/pending-payments/index.jsx

```diff
@@ -121,11 +121,14 @@ export default connect(
 
 		for ( const payment of data ) {
 			pending.push( {
-				siteId: payment.site_id,
 				orderId: payment.order_id,
+				siteId: payment.site_id,
+				paymentMethod: payment.payment_method,
 				paymentType: payment.payment_type,
 				redirectUrl: payment.redirect_url,
-				totalCostDisplay: payment.total_cost,
+				totalCost: payment.total_cost,
```

Comment:
I just noticed this may be wrong - in my case when I upgraded from one plan to another, I got a price reduction. The cost is shown in the pending payment was the full cost, not the one actually paid.

Reply:
May be fixed by D23668-code

Reply:
Definitely fixed by D23821-code